src/test/java/com/bieganski/ox/PositionValidatorTest.java

```diff
@@ -0,0 +1,60 @@
+package com.bieganski.ox;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class PositionValidatorTest {
```

Comment:
You could have more data to test here. Maybe try to generate valid/invalid strings automatically in great amounts?